lib/robots/dor_repo/gis_assembly/finish_gis_assembly_workflow.rb

```diff
@@ -26,7 +26,7 @@ def perform_work
           FileUtils.mkdir_p(destdir) unless File.directory?(destdir)
           logger.info "finish-gis-assembly-workflow: #{bare_druid} migrating object to #{destdir} from #{rootdir}"
           FileUtils.cp_r("#{rootdir}/.", destdir)
-          FileUtils.rm_f(rootdir)
+          FileUtils.rm_r(rootdir)
         end
```

Comment:
writing the test uncovered this bug ... `rm_f` only works on files, and not directories.. so I think this is not currently cleaning up root directories as it is expected to do